rest-service/manager_rest/resource_manager.py

```diff
@@ -550,12 +550,6 @@ def _reset_operations(self, execution, execution_token, from_states=None):
                     continue
 
                 update_operation = False
```

Comment:
`update_operation` can be removed and the `self.sm.update` can be back under the `if operation.state in from_states`